### PR TITLE
Marketing updates

### PR DESCRIPTION
**Pull request by yanncaniouoracle**
Marketing minor updates and typos. Moving from pip to conda env.

### CODE REVIEW

**Review by bruno-garbaccio — APPROVED** [review]

**Review by paulparkinson — APPROVED** [review]

**Review by bruno-garbaccio — DISMISSED** [review]

**Comment by AlexanderHodicke** [comment]
@oheimburger Could you please review the code - Thank you!

**Comment by jasperan** [comment]
hey any updates? We're planning to release this update tomorrow

**Review by paulparkinson — APPROVED** [review]

**Comment by jasperan** [comment]
@paulparkinson looks like you aren't an approved reviewer for this repo

**Review by enschilling — APPROVED** [review]

File: cloud-infrastructure/ai-infra-gpu/AI Infrastructure/rag-langchain-vllm-mistral/rag-langchain-vllm-mistral.py [hunk 1]
```diff
@@ -13,8 +13,8 @@
 documents = loader.load_data(
     sitemap_url='https://objectstorage.eu-frankfurt-1.oraclecloud.com/n/frpj5kvxryk1/b/thisIsThePlace/o/latest.xml'
```
Comment (oheimburger) [inline cloud-infrastructure/ai-infra-gpu/AI Infrastructure/rag-langchain-vllm-mistral/rag-langchain-vllm-mistral.py:14]: No real URLs, please.

File: cloud-infrastructure/ai-infra-gpu/AI Infrastructure/nim-gpu-oke/README.md [hunk 2]
```diff
@@ -1,281 +1,285 @@
...
 
-You can then see the Public IP of you grafana dashboard by running:
+You can then see the Public IP of your Grafana dashboard by running:
 
-```
+```bash
 $ kubectl get svc
 NAME                                       TYPE           CLUSTER-IP     EXTERNAL-IP       PORT(S)                      AGE
 alertmanager-operated                      ClusterIP      None           <none>            9093/TCP,9094/TCP,9094/UDP   2m33s
 example-metrics-grafana                    LoadBalancer   10.96.82.33    141.145.220.114   80:31005/TCP                 2m38s
```
Comment (oheimburger) [inline cloud-infrastructure/ai-infra-gpu/AI Infrastructure/nim-gpu-oke/README.md:222]: No real IPs, please.

File: cloud-infrastructure/ai-infra-gpu/AI Infrastructure/nim-gpu-oke/README.md [hunk 3]
```diff
@@ -1,281 +1,285 @@
...
 
-Now that the inference server is running you can send HTTP or GRPC requests to it to perform inferencing. By default, the inferencing service is exposed with a LoadBalancer service type. Use the following to find the external IP for the inference server. In this case it is 34.83.9.133.
+Now that the inference server is running you can send HTTP or GRPC requests to it to perform inferencing. By default, the inferencing service is exposed with a LoadBalancer service type. Use the following to find the external IP for the inference server. In this case, it is 34.83.9.133.
 
-```
+```bash
 $ kubectl get services
 NAME                             TYPE           CLUSTER-IP     EXTERNAL-IP   PORT(S)                                        AGE
 ...
 example-triton-inference-server  LoadBalancer   10.18.13.28    34.83.9.133   8000:30249/TCP,8001:30068/TCP,8002:32723/TCP   47m
```
Comment (oheimburger) [inline cloud-infrastructure/ai-infra-gpu/AI Infrastructure/nim-gpu-oke/README.md:257]: No real IPs, please.

File: cloud-infrastructure/ai-infra-gpu/AI Infrastructure/nim-gpu-oke/README.md [hunk 4]
```diff
@@ -1,281 +1,285 @@
...
 ```
 
-The inference server exposes an HTTP endpoint on port 8000, and GRPC endpoint on port 8001 and a Prometheus metrics endpoint on port 8002. You can use curl to get the meta-data of the inference server from the HTTP endpoint.
+The inference server exposes an HTTP endpoint on port 8000, and GRPC endpoint on port 8001, and a Prometheus metrics endpoint on port 8002. You can use curl to get the meta-data of the inference server from the HTTP endpoint.
 
-```
+```bash
 $ curl 34.83.9.133:8000/v2
```
Comment (oheimburger) [inline cloud-infrastructure/ai-infra-gpu/AI Infrastructure/nim-gpu-oke/README.md:263]: No real IPs, please.

**Review by oheimburger — COMMENTED** [review]
Please address the issues highlighted. You need to delete this PR and create a new sanitized PR.

**Pull request merged**